app/views/devise/sessions/new.slim

```diff
@@ -1,25 +1,19 @@
 .container
+  .divide80
   .row
-    .col-md-12
+    .col-md-6
       .row
-        .col-md-2.col-md-offset-0.col-sm-8.col-sm-offset-2.col-xs-8.col-xs-offset-2.sei-logo-sign-in
+        .col-md-10
           = image_tag "sei-logo.svg", class: "responsive-image"
-      .form-container
-        h2 Log in
-        = form_for(resource, as: resource_name, url: session_path(resource_name)) do |f|
-          .field
-            = f.label :email
-            br
-            = f.email_field :email, autofocus: true
-          .field
-            = f.label :password
-            br
-            = f.password_field :password, autocomplete: "off"
-          // - if devise_mapping.rememberable?
-          //   .field
-          //     = f.check_box :remember_me
-          //     = f.label :lembrar_login
-          = button_tag "Log in", type: 'submit',
-            class: 'btn btn-submit col-md-2 col-xs-12', id: 'user-new-login'
-          = link_to "Registar", new_user_registration_path,
-            class: 'btn btn-submit col-md-2 col-xs-12'
+    .col-md-6
+      .divide50
+      .login-form
+        h2 Login
+        = link_to "/auth/google_oauth2",
```

Comment:
You should prepend `/users` because of the routes defined.

Reply:
Copied the link from atomic and forgot to change the paths :S